CLAUDE.md

```
@@ -61,21 +61,21 @@ The project uses **StyleCI** with PSR-2 preset for code style enforcement. Style
 ## Modern Architecture Components
 
 ### Database Layer (`/src/Database/`)
-- **Nette Database** with full old SqlDb backward compatibility
-- Singleton pattern accessible via `DB()` function
+- **Nette Database** replacing legacy SqlDb system
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)